src/app/components/dashboard/dashboard.controller.js

```diff
@@ -24,5 +27,47 @@
       });
     });
 
+    $scope.removeFavorite = function (list) {
+      $scope.currentUser.favoriteLists.splice($scope.currentUser.favoriteLists.indexOf(list), 1);
+
+      User.update($scope.currentUser, function () {
+        alertService.add('success', gettextCatalog.getString('This list was removed from your favorites.'));
+        $scope.setCurrentUser($scope.currentUser);
+      });
+    };
+
+    $scope.leaveList = function (list) {
```

Comment:
@guillaumev I've copied and adapted this from how you remove a user from a list in the list controller, is this the correct way to do it?